src/ImageSharp.Web/Commands/CommandCollectionExtensions.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Collections.Generic;
+
+namespace SixLabors.ImageSharp.Web.Commands
+{
+    /// <summary>
+    /// Extension methods for <see cref="CommandCollectionExtensions"/>.
+    /// </summary>
+    public static class CommandCollectionExtensions
+    {
+        /// <summary>
+        /// Gets the value associated with the specified key or the default value.
+        /// </summary>
+        /// <param name="collection">The collection instance.</param>
+        /// <param name="key">The key of the value to get.</param>
+        /// <returns>The value associated with the specified key or the default value.</returns>
+        public static string GetValueOrDefault(this CommandCollection collection, string key)
```

Comment:
I didn't see this being used anywhere

Do you really want it? Wouldn't just using `TryGetValue`  be enough / better?

Reply:
It’s used a lot in the individual processors. Allows much less code when passing values to the command parser which already handles the default instance

Reply:
Don't forget to remove the `DictionaryExtensions` class if that becomes obsolete by this change!

And if my suggestion regarding adding a `TryGetValue(string key, out string value)` overload doesn't get included, this makes even more sense, as you don't need the `KeyValuePair<string, string>` if you already know the key...